onmt/opts.py

```diff
@@ -244,6 +245,12 @@ def model_opts(parser):
     group.add('--position_encoding', '-position_encoding', action='store_true',
               help="Use a sin to mark relative words positions. "
                    "Necessary for non-RNN style models.")
+    group.add('--position_encoding_type', '-position_encoding_type',
+              type=str, default='SinusoidalInterleaved',
+              choices=['SinusoidalInterleaved', 'SinusoidalConcat'],
+              help="Type of positional encoding. At the moment: "
+                   "Sinusoidal fixed, Interleaved or Concat")
```

Comment:
Do we have some refs (papers) for these methods?

Reply:
not really. just different implementations. cf https://stackoverflow.com/questions/63295569/explanation-about-i-2-in-positional-encoding-in-tensorflow-tutorial-about-trans